### ISSUE TITLE
Panic during analysis

### ISSUE THREAD

**Issue by kenjones-cisco** [tetafro/godot#10]
if there is a comment line that looks like the following:
```
// )
```
Then within `checkLastChar` the close parenthesis is removed here (https://github.com/tetafro/godot/blob/ac903ab935ebe430ccf89bd2aa4630ca89640f50/godot.go#L293), such that when the line is checked, it results in a panic due to index out of range. (see below)

It seems a check for `if s == ""` is missing to avoid the scenario. 

https://github.com/tetafro/godot/blob/ac903ab935ebe430ccf89bd2aa4630ca89640f50/godot.go#L295
```
level=warning msg="[linters context] Panic: godot: package \"service\" (isInitialPkg: true, needAnalyzeSource: true): runtime error: index out of range [-1]: goroutine 14590 [running]:
runtime/debug.Stack(0xf1ea64, 0x3c, 0xc021397368)
	/opt/hostedtoolcache/go/1.15.0/x64/src/runtime/debug/stack.go:24 +0x9f
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyzeSafe.func1(0xc0022cd0e0)
	/home/runner/work/golangci-lint/golangci-lint/pkg/golinters/goanalysis/runner.go:508 +0x1be
panic(0xe0ce60, 0xc001946800)
	/opt/hostedtoolcache/go/1.15.0/x64/src/runtime/panic.go:969 +0x175
github.com/tetafro/godot.checkLastChar(0xc01ef29d07, 0x0, 0x2)
	/home/runner/go/pkg/mod/github.com/tetafro/godot@v0.4.8/godot.go:295 +0x4ff
github.com/tetafro/godot.checkComment(0xc01ef29d04, 0x4, 0x1, 0xc001c5e660, 0x21)
	/home/runner/go/pkg/mod/github.com/tetafro/godot@v0.4.8/godot.go:235 +0xc9
github.com/tetafro/godot.check(0xc0017de040, 0xc01ef4ba00, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/home/runner/go/pkg/mod/github.com/tetafro/godot@v0.4.8/godot.go:211 +0x14a
github.com/tetafro/godot.checkDeclarations(0xc019df6480, 0xc0017de040, 0x0, 0x0, 0x0)
	/home/runner/go/pkg/mod/github.com/tetafro/godot@v0.4.8/godot.go:160 +0x105
github.com/tetafro/godot.Run(0xc019df6480, 0xc0017de040, 0x0, 0xc0122b0d80, 0x5, 0x5)
	/home/runner/go/pkg/mod/github.com/tetafro/godot@v0.4.8/godot.go:71 +0x1dd
github.com/golangci/golangci-lint/pkg/golinters.NewGodot.func1.1(0xc02ee2a900, 0xc6c50da85, 0x165a860, 0xc015979630, 0x2)
	/home/runner/work/golangci-lint/golangci-lint/pkg/golinters/godot.go:36 +0x14d
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyze(0xc0022cd0e0)
	/home/runner/work/golangci-lint/golangci-lint/pkg/golinters/goanalysis/runner.go:590 +0xa83
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyzeSafe.func2()
	/home/runner/work/golangci-lint/golangci-lint/pkg/golinters/goanalysis/runner.go:512 +0x2a
github.com/golangci/golangci-lint/pkg/timeutils.(*Stopwatch).TrackStage(0xc0016da640, 0xe705ce, 0x5, 0xc00261a770)
	/home/runner/work/golangci-lint/golangci-lint/pkg/timeutils/stopwatch.go:111 +0x50
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyzeSafe(0xc0022cd0e0)
	/home/runner/work/golangci-lint/golangci-lint/pkg/golinters/goanalysis/runner.go:511 +0x91
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*loadingPackage).analyze.func2(0xc005493dd0, 0xc0022cd0e0)
	/home/runner/work/golangci-lint/golangci-lint/pkg/golinters/goanalysis/runner.go:1059 +0x65
created by github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*loadingPackage).analyze
	/home/runner/work/golangci-lint/golangci-lint/pkg/golinters/goanalysis/runner.go:1054 +0x305
```

**Comment by tetafro** [comment]
Thanks for detailed report. I'll fix it.

**Comment by tetafro** [comment]
Fixed in 0.4.9.

dependabot will update the package in golangci-lint in few days.
